### ISSUE TITLE
[Bug]: Using Collecting Event Radial to Change Bulk Geographic Areas gives 504 Bad Gateway timeout error

### ISSUE THREAD

**Issue by tmcelrath** [SpeciesFileGroup/taxonworks#3527]
### Steps to reproduce the bug

```bash
1. When I am in "Filter Collecting Events"
2. And I select many records (>1000)
3. And I use Radial Collecting Event to "Assign/Move" them to a new Geographic Area.
4. And I want a long time.
5. Eventually I get a 504 Bad Gateway timeout error.
6. But I can see them processing in the background if I redo the filter search. 

Works for up to 250 records, fails at 500 records.
```

### Screenshot

![badtimeout](https://github.com/SpeciesFileGroup/taxonworks/assets/11411328/efd2fc78-4812-4831-99b2-9d23293e6072)

### Expected behavior

I would expect to be able to process >1000 records without timeout errors. In fact, I want to be able to do 20k records without a timeout error. I don't care about the time, I just want to see the process. 

Having a XXX records/XXXXX records progress bar that refreshes every 10 - 30 seconds would be fine. 

### Environment

Production

### Version

0.34.2

### Browser Used

Chrome

**Comment by mjy** [comment]
@LocoDelAssembly could this be POST size limit issue with gateway?

**Comment by LocoDelAssembly** [comment]
No, this is fails because the backend takes more than 10 minutes to send the first byte of a response. Timeout is already high so this needs to be optimized/paged/etc.

**Comment by LocoDelAssembly** [comment]
@tmcelrath if `4. And I want a long time.` takes less than 10 minutes approximately please let us know.

**Comment by tmcelrath** [comment]
The job is still running, even after timeout, I've confirmed. 

So far ~10000 records was started ~2 hours ago, only through about 4000 records.

**Comment by tmcelrath** [comment]
Timeout error appeared after a few minutes of starting.
